supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -11,5 +11,31 @@
 [[admonitions]]
 == Admonitions
 
+Admonitions should draw the reader’s attention to certain information. Keep admonitions to a minimum, and avoid placing multiple admonitions in close proximity to one another. If multiple admonitions are necessary, restructure the information by removing the less important statements and placing them into the flow of the main content.
+
+Allowed admonition types:
+
+- NOTE
+- IMPORTANT
+- WARNING
+- TIP
+
+[IMPORTANT]
+====
+CAUTION, which is another type of AsciiDoc admonition, is not fully supported by the Red Hat Customer Portal. This admonition should not be used.
+====
+
+Admonitions should be short and concise. Do not include procedures in an admonition.
+
+Only individual admonitions are allowed. For example, you cannot have a plural "Notes" heading.
```

Comment:
Avoid using quotation marks in this context. Suggestion: `you cannot have a plural *NOTES* heading.`

Reply:
We shouldn't be using bold except for GUI items. Do you have another suggestion?

Reply:
@bergerhoffer  @stoobie I can change it to the following: ` you cannot have a plural _NOTES_ heading.`
What do you think?

Reply:
> We shouldn't be using bold except for GUI items. Do you have another suggestion?

@bergerhoffer  Is that in the IBM Style Guide? So italics can be used for doc titles, user-replaced variables, emphasis, new terms...? Is that right?

Reply:
Sorry @stoobie I think my feedback there leaked in from a rule we follow on OpenShift. I don't think that we have a blanket bold-only-GUI-items statement in either the IBM style guide or our supplementary style guide.

@lmalivert Sorry to point you in the wrong direction, it should be fine to use bold for **NOTES** here as @stoobie suggested.

Reply:
@bergerhoffer No problem. I appreciate everyone's feedback. I'll bold **NOTES** as @stoobie suggested.